src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -123,6 +128,61 @@ export const NoMatchingResults = (): JSX.Element => {
     return <GenericFilterEmptyState />
 }
 
+const ExternalLinkIframeModal = ({ selectedExternalLink, handleCloseModal }) => {
+    const handleOpenExternalLink = () => {
+        window.open(selectedExternalLink.externalLinkURL, '_blank')
+    }
+
+    return (
+        <VisibleModal2 close={handleCloseModal}>
+            <div
+                className="flexbox-col dc__position-abs br-8 dc__top-12 dc__bottom-12 dc__right-12 dc__left-12 bcn-0"
+                onClick={stopPropagation}
+            >
+                <div className="flexbox dc__content-space px-20 py-12 dc__align-items-center dc__border-bottom">
+                    <div className="flexbox dc__gap-8 dc__align-items-center">
+                        <img
+                            className="icon-dim-20"
+                            src={selectedExternalLink.icon}
+                            alt={selectedExternalLink.label}
+                            onError={onImageLoadError}
+                        />
+                        <span className="cn-9 fs-16 fw-6 lh-24">{selectedExternalLink.label}</span>
+                    </div>
+                    <div className="flexbox dc__gap-8">
+                        <Button
+                            ariaLabel="external-link-open"
+                            dataTestId="external-link-open"
+                            icon={<ICArrowOut className="scn-6" />}
```

Comment:
className is handled inside button itself

Reply:
Icon is transparent, no stroke is visible without scn